defaults/test_default.yaml

```diff
@@ -112,7 +112,7 @@ authenticator_password: ''
 # gemini defaults
 n_test_oracle_db_nodes: 1
 gemini_seed: 0
-oracle_scylla_version: '2021.1.23'
+oracle_scylla_version: '2023.1'
```

Comment:
better to set exact version 2023.1.1 or the release lastest one

Reply:
2023.1 is not good test oracle because in most cases it just comparing one to itself.
The patches still flowing from master, enterprise to 2024.1 and 2023.1.
It means that if there is a regression we recently introduce to 2023.1 all newer releases will suffer the same.

Reply:
Maybe we can change to 2022.1.20 (or something like that).

Reply:
but there is no 2022.2.20 image 
```
artsiom@artsiom-mishuta-Dell-G15-Special-Edition-5521:~/codebase/scylla-cluster-tests$ hydra list-images -v 2022.2.20
There is scylladb/hydra:v1.62-cryptography-42.0.4 in local cache, using it.
Going to run './sct.py  list-images -v 2022.2.20'...
logged in as arn:aws:iam::797456418907:user/artsiom.mishuta
New directory created: /home/artsiom/sct-results/20240506-130230-031500-list-images
+--------------------------------------------------------+
|   AWS Machine Images by Version in region eu-west-1    |
+----------+-------+----------+---------------+----------+
| Backend  | Name  | ImageId  | CreationDate  | NameTag  |
+----------+-------+----------+---------------+----------+
+----------+-------+----------+---------------+----------+
```
image, only 19  
```
artsiom@artsiom-mishuta-Dell-G15-Special-Edition-5521:~/codebase/scylla-cluster-tests$ hydra list-images -v 2022.2.19
There is scylladb/hydra:v1.62-cryptography-42.0.4 in local cache, using it.
Going to run './sct.py  list-images -v 2022.2.19'...
logged in as arn:aws:iam::797456418907:user/artsiom.mishuta
New directory created: /home/artsiom/sct-results/20240506-130312-160907-list-images
+------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                  AWS Machine Images by Version in region eu-west-1                                                   |
+---------+-------------------------------+-----------------------+--------------------------+---------------------------------------------------------+
| Backend | Name                          | ImageId               | CreationDate             | NameTag                                                 |
+---------+-------------------------------+-----------------------+--------------------------+---------------------------------------------------------+
| AWS     | ScyllaDB Enterprise 2022.2.19 | ami-07dcbdfe4d03fe54f | 2024-04-30T05:22:29.000Z | scylla-enterprise-2022.2.19-x86_64-2024-03-28T11-20-35Z |
+---------+-------------------------------+-----------------------+--------------------------+---------------------------------------------------------+

```

Reply:
Check 2022.1

Reply:
hydra list-images -v 2022.1
There is scylladb/hydra:v1.62-cryptography-42.0.4 in local cache, using it.
Going to run './sct.py  list-images -v 2022.1'...
logged in as arn:aws:iam::797456418907:user/artsiom.mishuta
New directory created: /home/artsiom/sct-results/20240507-100953-585642-list-images
+------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                  AWS Machine Images by Version in region eu-west-1                                                   |
+---------+-------------------------------+-----------------------+--------------------------+---------------------------------------------------------+
| Backend | Name                          | ImageId               | CreationDate             | NameTag                                                 |
+---------+-------------------------------+-----------------------+--------------------------+---------------------------------------------------------+
| AWS     | ScyllaDB Enterprise 2022.1.14 | ami-00cadd5455ec9e7a8 | 2024-02-04T06:17:13.000Z | scylla-enterprise-2022.1.14-x86_64-2024-01-24T13-11-36Z |
| AWS     | ScyllaDB Enterprise 2022.1.13 | ami-009dfabeb2a479730 | 2023-12-20T12:43:12.000Z | scylla-enterprise-2022.1.13-x86_64-2023-12-12T00-58-31Z |
| AWS     | ScyllaDB Enterprise 2022.1.12 | ami-0a56e98a5d31d14ec | 2023-12-04T21:16:59.000Z | scylla-enterprise-2022.1.12-x86_64-2023-11-30T07-20-52Z |
| AWS     | ScyllaDB Enterprise 2022.1.11 | ami-0616c533f2866dbb8 | 2023-10-15T11:24:09.000Z | scylla-enterprise-2022.1.11-x86_64-2023-09-11T20-59-05Z |
| AWS     | ScyllaDB Enterprise 2022.1.10 | ami-011bdbb5130800bd5 | 2023-09-03T07:55:48.000Z | scylla-enterprise-2022.1.10-x86_64-2023-08-21T01-09-07Z |
| AWS     | ScyllaDB Enterprise 2022.1.9  | ami-063c4227a286b3d85 | 2023-08-06T12:11:03.000Z | scylla-enterprise-2022.1.9-x86_64-2023-07-27T05-32-25Z  |
| AWS     | ScyllaDB Enterprise 2022.1.8  | ami-0da1051800e5a4ae6 | 2023-07-11T16:38:55.000Z | scylla-enterprise-2022.1.8-x86_64-2023-07-06T10-31-07Z  |
| AWS     | ScyllaDB Enterprise 2022.1.7  | ami-06af967f13bbe9d66 | 2023-06-15T12:16:21.000Z | scylla-enterprise-2022.1.7-x86_64-2023-06-05T23-50-06Z  |
| AWS     | ScyllaDB Enterprise 2022.1.6  | ami-008b8ab71e03eb539 | 2023-05-08T07:11:27.000Z | scylla-enterprise-2022.1.6-x86_64-2023-04-20T00-21-14Z  |
| AWS     | ScyllaDB Enterprise 2022.1.5  | ami-0287067067652acb9 | 2023-01-12T07:41:38.000Z | scylla-enterprise-2022.1.5-x86_64-2023-01-08T11-50-09Z  |
| AWS     | ScyllaDB Enterprise 2022.1.4  | ami-01af264a642b89a1f | 2023-01-12T07:36:26.000Z | scylla-enterprise-2022.1.4-x86_64-2022-12-19T07-45-56Z  |
| AWS     | ScyllaDB Enterprise 2022.1.3  | ami-0f6aebcffc8f7aa66 | 2022-10-12T15:16:10.000Z | scylla-enterprise-2022.1.3-x86_64-2022-09-22T15-59-20Z  |
| AWS     | ScyllaDB Enterprise 2022.1.2  | ami-0ba5ceb867818a6ad | 2022-09-18T12:15:43.000Z | scylla-enterprise-2022.1.2-x86_64-2022-09-06T01-18-56Z  |
| AWS     | ScyllaDB Enterprise 2022.1.1  | ami-0fe69824e533e91f4 | 2022-08-24T11:21:00.000Z | Packer Builder                                          |
| AWS     | ScyllaDB Enterprise 2022.1.0  | ami-09ff93f1dff1ca9af | 2022-08-02T07:27:42.000Z | Packer Builder                                          |
+---------+-------------------------------+-----------------------+--------------------------+---------------------------------------------------------+

+----------+-------+----------+---------------+----------+

Reply:
BTW 2022.2.19 passed https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/artsiom_mishuta/job/sct_gemini_tests/job/gemini-3h-ics-test/15/

Reply:
@roydahan should I change/check against 2022.1.14 ?

Reply:
Yes, Just change to it.
It’s a release we hardly backport to anymore.